person/src/com/person/entrypoint/Main.java

```diff
@@ -0,0 +1,58 @@
+package com.person.entrypoint;
+
+import com.person.core.containers.PersonList;
+import com.person.core.enumeration.Profession;
+import com.person.core.model.BasePerson;
+import com.person.core.service.factory.PersonFactory;
+import com.person.utils.Input;
+import com.person.utils.PersonData;
+
+public class Main {
+    public static void main(String[] args)
+    {
+        PersonList people = new PersonList();
+        BasePerson person = null;
+
+        int errorCount = 0;
+        while(errorCount < 3) {
+            System.out.println("~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~");
+            String command = Input.string("Command");
+
+            if(command.equals("new")) {
```

Comment:
@tatevikb you can't keep such a big method please divide it on small units also this is a good example where you need comments